cmd/feature.go

```diff
@@ -0,0 +1,194 @@
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/cozy/cozy-stack/client/request"
+	"github.com/spf13/cobra"
+)
+
+var featureCmdGroup = &cobra.Command{
+	Use:   "feature <command>",
```

Comment:
`feature` in singular is not coherent with `instances` in plural.
At least call it `features` and add an `feature` alias